mmv1/third_party/terraform/website/docs/r/compute_firewall_policy_rule.html.markdown

```diff
@@ -0,0 +1,162 @@
+  
+* `match` -
+  (Required)
+  A match condition that incoming traffic is evaluated against. If it evaluates to true, the corresponding 'action' is enforced.
+  
+* `priority` -
+  (Required)
+  An integer indicating the priority of a rule in the list. The priority must be a positive value between 0 and 2147483647. Rules are evaluated from highest to lowest priority where 0 is the highest priority and 2147483647 is the lowest prority.
+  
+
+
+The `match` block supports:
+    
+* `dest_ip_ranges` -
+  (Optional)
+  CIDR IP address range. Maximum number of destination CIDR IP ranges allowed is 256.
+    
+* `layer4_configs` -
+  (Required)
+  Pairs of IP protocols and ports that the rule should match.
+    
+* `src_ip_ranges` -
+  (Optional)
+  CIDR IP address range. Maximum number of source CIDR IP ranges allowed is 256.
+    
+The `layer4_configs` block supports:
+    
+* `ip_protocol` -
+  (Required)
+  The IP protocol to which this rule applies. The protocol type is required when creating a firewall rule. This value can either be one of the following well known protocol strings (`tcp`, `udp`, `icmp`, `esp`, `ah`, `ipip`, `sctp`), or the IP protocol number.
+    
+* `ports` -
+  (Optional)
+  An optional list of ports to which this rule applies. This field is only applicable for UDP or TCP protocol. Each entry must be either an integer or a range. If not specified, this rule applies to connections through any port. Example inputs include: ``.
+    
+- - -
+
+* `description` -
+  (Optional)
+  An optional description for this resource.
+  
+* `disabled` -
+  (Optional)
+  Denotes whether the firewall policy rule is disabled. When set to true, the firewall policy rule is not enforced and traffic behaves as if it did not exist. If this is unspecified, the firewall policy rule will be enabled.
+  
+* `enable_logging` -
+  (Optional)
+  Denotes whether to enable logging for a particular rule. If logging is enabled, logs will be exported to the configured export destination in Stackdriver. Logs may be exported to BigQuery or Pub/Sub. Note: you cannot enable logging on "goto_next" rules.
+  
+* `target_resources` -
+  (Optional)
+  A list of network resource URLs to which this rule applies. This field allows you to control which network's VMs get this rule. If this field is left blank, all VMs within the organization will receive the rule.
+  
+* `target_service_accounts` -
+  (Optional)
+  A list of service accounts indicating the sets of instances that are applied with this rule.
+  
+
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `id` - an identifier for the resource with format `locations/global/firewallPolicies/{{firewall_policy}}/rules/{{priority}}`
+
+* `kind` -
+  Type of the resource. Always `compute#firewallPolicyRule` for firewall policy rules
+  
+* `rule_tuple_count` -
+  Calculation of the complexity of a single firewall policy rule.
```

Comment:
non-blocking: https://cloud.google.com/compute/docs/reference/rest/v1/firewallPolicies has _way_ clearer documentation for this field. Is the DCL documentation all provided based on an old version of the docs?

Reply:
Where do you see better documentation? I see two different versions of `ruleTupleCount` on that page with different descriptions, but neither are particularly great

Reply:
Oh, nvm- I saw the description for the top-level version and it made more sense than this one. Both are in sync with the page.